.envs/environment.yml

```diff
@@ -0,0 +1,15 @@
+name: klaam
+channels:
+  - defaults
+  - conda-forge
+  - pytorch
+  - huggingface
+dependencies:
+  - python=3.8
+  - poetry
+  # klaam dependencies
+  - librosa
+  - ffmpeg
+  - cudatoolkit
+  - pytorch
+  - torchaudio
```

Comment:
I think soundfile package is missing?

Reply:
I think `soundfile` is a dependency of another package (maybe `torchaudio`) that is installed by default. I have managed to run `notebooks/demo.ipynb` without installing `soundfile` explicitly.

To give more context, `.env/dev_environment.yml` contains the same packages as `.env/environment.yml` but CPU version so we can run CI on GitHub servers.

Reply:
You are right about soundfile [https://pytorch.org/audio/stable/backend.html]